lib/nerves_network.ex

```diff
@@ -26,11 +33,19 @@ defmodule Nerves.Network do
           | {:ipv4_subnet_mask, Types.ip_address()}
           | {:domain, String.t()}
           | {:nameservers, [Types.ip_address()]}
-          | {atom, any()}
+          | {:networks, [[wifi_settings]]}
```

Comment:
I think the formatter wants:

```suggestion
          | {:networks, [[wifi_settings()]]}
```

Reply:
when i run `mix format --check-formatted` it doesn't complain about it, but i do think i like that better

Reply:
Maybe they've changed the rules since I last got it burned into my brain. :)